src/screens/Assets/PPNView.js

```diff
@@ -228,23 +228,23 @@ class PPNView extends React.Component<Props, State> {
       }
 
       return null;
-    } else if (archanovaWalletStatus.status === SMART_WALLET_UPGRADE_STATUSES.ACCOUNT_CREATED) {
+    } else if (archanovaWalletStatus.status === ARCHANOVA_WALLET_UPGRADE_STATUSES.ACCOUNT_CREATED) {
       return (
         <InsightWithButton
           title={t('insight.pillarNetworkActivate.smartWalletIsNotActivated.title')}
           itemsList={[
             t('insight.pillarNetworkActivate.smartWalletIsNotActivated.description.instantTransactions'),
             t('insight.pillarNetworkActivate.smartWalletIsNotActivated.description.singleTokenExperience')]}
           buttonTitle={t('insight.pillarNetworkActivate.smartWalletIsNotActivated.button.activatePPN')}
-          onButtonPress={this.openSmartWalletModal}
+          onButtonPress={this.openArchanovaActivationModal}
         />
       );
     }
 
     return null;
   };
 
-  openSmartWalletModal = () => {
+  openArchanovaActivationModal = () => {
     Modal.open(() => <SWActivationModal navigation={this.props.navigation} />);
   };
```

Comment:
```suggestion
  openArchanovaActivationModal = () => Modal.open(() => <SWActivationModal navigation={this.props.navigation} />);
```

Totally optional though.